python/src/ConditionalDistribution_doc.i.in

```diff
@@ -39,6 +39,8 @@ With the default constructor, the resulting random variable :math:`X` follows a
 
 Note that a :class:`~openturns.ConditionalDistribution` can be obtained as the first marginal distribution of a :class:`~openturns.BayesDistribution`.
 
+The continuous part of the conditioning distribution is discretized using a :class:`~openturns.WeightedExperiment` which can be modified using 'ConditionalDistribution-ContinuousDiscretizationMethod' in :class:`~openturns.ResourceMap` (can be 'GaussProduct', 'QMC' or 'MC'), in conjunction with 'ConditionalDistribution-MarginalIntegrationNodesNumber' and 'ConditionalDistribution-MaximumIntegrationNodesNumber'. The default is 'GaussProduct', well adapted if the conditioned model is *regular*, meaning that its range does not depend on the parameters (e.g. a :class:`~openturns.Normal` distribution). Otherwise, one of the other discretization methods is to be prefered (e.g. for a :class:`~openturns.Uniform` distribution).
```

Comment:
```suggestion
The continuous part of the conditioning distribution is discretized using a :class:`~openturns.WeightedExperiment` which can be modified using 'ConditionalDistribution-ContinuousDiscretizationMethod' in :class:`~openturns.ResourceMap` (can be 'GaussProduct', 'QMC' or 'MC'), in conjunction with 'ConditionalDistribution-MarginalIntegrationNodesNumber' and 'ConditionalDistribution-MaximumIntegrationNodesNumber'. The default is 'GaussProduct', well adapted if the conditioned model is *regular*, meaning that its range does not depend on the parameters (e.g. a :class:`~openturns.Normal` distribution). Otherwise, one of the other discretization methods is to be preferred (e.g. for a :class:`~openturns.Uniform` distribution).
```